frontend/src/metabase/visualizations/components/List/VariantInfo.tsx

```diff
@@ -24,50 +27,69 @@ export const VariantInfo = ({
     return null;
   }
 
+  const [imageIndex] = listColumnIndexes.image;
   const [titleIndex, subtitleIndex, subtitle2Index] = listColumnIndexes.left;
   const [infoIndex] = listColumnIndexes.right;
 
+  const image = row[imageIndex];
   const title = row[titleIndex];
   const subtitle = row[subtitleIndex];
   const subtitle2 = row[subtitle2Index];
-
   const info = row[infoIndex];
 
+  const imageColIsImage = isImageURL(data.cols[imageIndex]);
+
   return (
     <InfoListItem>
       <InfoLeft>
-        <ListItemTitle>
-          {!isEmpty(title) && (
-            <ListCell
-              value={title}
-              columnTitle={getColumnTitle(titleIndex)}
-              data={data}
-              settings={settings}
-              columnIndex={titleIndex}
-            />
-          )}
-        </ListItemTitle>
-        <ListItemSubtitle>
-          {!isEmpty(subtitle) && (
-            <ListCell
-              value={subtitle}
-              columnTitle={getColumnTitle(subtitleIndex)}
-              data={data}
-              settings={settings}
-              columnIndex={subtitleIndex}
-            />
-          )}
-          {!isEmpty(subtitle) && !isEmpty(subtitle2) && " - "}
-          {!isEmpty(subtitle2) && (
-            <ListCell
-              value={subtitle2}
-              columnTitle={getColumnTitle(subtitle2Index)}
-              data={data}
-              settings={settings}
-              columnIndex={subtitle2Index}
-            />
-          )}
-        </ListItemSubtitle>
+        {image && imageColIsImage && (
+          <ListCell
+            value={image}
+            columnTitle={getColumnTitle(imageIndex)}
+            data={data}
+            settings={settings}
+            columnIndex={imageIndex}
+          />
+        )}
+        {image && !imageColIsImage && (
+          <Avatar>
+            {String(row?.[imageIndex])?.slice(0, 1)?.toUpperCase()}
```

Comment:
I think it's worth extracting into a function. I assume it's extracting first letters to show initials, but it might not be immediately clear